dev-libs/libp11/libp11-0.4.3.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="A library that implements a layer on top of PKCS11 API"
```

Comment:
Maybe
```
SUMMARY="A PKCS#11 wrapper library"
```
would be better, to avoid the repetition with the suggestion for DESCRIPTION.